src/utils/api/wrapper/shoppingListEndpoints.ts

```diff
@@ -1,14 +1,55 @@
-import { BASE_URI, combinedHeaders } from './shared'
+import { type RequestShoppingList } from '../../../types/apiData'
+import { BASE_URI, combinedHeaders } from '../sharedUtils'
 import {
+  type PostShoppingListsResponse,
+  type PostShoppingListsReturnValue,
   type GetShoppingListsResponse,
   type GetShoppingListsReturnValue,
 } from '../returnValues/shoppingLists'
 import {
   AuthorizationError,
   InternalServerError,
   NotFoundError,
+  UnprocessableEntityError,
 } from '../apiErrors'
 
+/**
+ *
+ * POST /games/:game_id/shopping_lists endpoint
+ *
+ */
+
+export const postShoppingLists = (
```

Comment:
I'm not satisfied with the order of the arguments to this function. However, I considered a different order and wasn't satisfied with that either. I know `token` comes last but I can't decide if `gameId` or `attributes` come first. I'm just going to call it good enough.